__integration_tests__/__tests__/coverage_report-test.js

```diff
@@ -0,0 +1,9 @@
+jest.unmock('../integrationUtil');
+const runJest = require('../integrationUtil').runJest;
+describe('coverage_report', () => {
```

Comment:
please add some newlines around here. After unmock and after the const assignment.

Also, you should do this:

```js
let runJest;

describe('Coverage Report', () => {
  beforeEach(() => {
    runJest = require('runJest');
  });
});
```

because we wipe away the entire module registry on every call to `it`, we'll want to re-require the module for every test as well so we don't use the same function for every test.